sig-scheduling/annual-report-2022.md

```diff
@@ -33,29 +35,29 @@
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
+   - increase the number of reviewers
```

Comment:
you can link to https://groups.google.com/g/kubernetes-sig-scheduling/c/ZzxAujy6dLM

Reply:
added